### PR TITLE
Update manubot

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/greenelab/meta-review/issues/4

### CODE REVIEW

**Comment by dhimmel** [comment]
Here's the PDF from building https://github.com/greenelab/meta-review/pull/5/commits/dd689a0a4e4e69bafa9f1dc7913fc7ff4677f8ac locally: [manuscript.pdf](https://github.com/greenelab/meta-review/files/1222492/manuscript.pdf)

File: README.md [hunk 1]
```diff
@@ -10,10 +10,39 @@ To see what's incoming, check the [open pull requests](https://github.com/greene
 
 This manuscript was build with [Manubot Rootstock](https://git.io/vQSvo), which provides a system for collaboratively writing scholarly manuscripts via GitHub.
 
-## Source
+Instructions for using Manubot Rootstock for your own manuscript are still evolving.
```
Comment (agitter) [inline README.md]: We could cut these two lines about cloning Manubot Rootstock and refer potential users back to the manubot-rootstock repository for the latest instructions.

File: content/02.main-text.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ Open research – which includes sharing code, data, and manuscripts – benefit
 Here we describe the benefits of writing review articles openly, where the planning, organizing, writing, and editing occur collaboratively in a public forum where participants are free to join as they wish.
 Reviews presenting the state of the art in a scientific field are often prepared by a single research group or a small team of colleagues.
 In contrast, broadly opening the writing process to anyone engaged in the topic can help maximize the review's value by facilitating the representation of diverse opinions and the broad coverage of relevant research.
-Review authors can engage with the authors of original research to clarify their methods and results and present them accurately (for example, [@url:https://github.com/greenelab/deep-review/issues/213]).
+Review authors can engage with the authors of original research to clarify their methods and results and present them accurately (for example, @url:https://github.com/greenelab/deep-review/issues/213).
```
Comment (agitter) [inline content/02.main-text.md:10]: Why does this URL not have `[ ]` but the others do?  The PDF shows it as `[4]`.

**Review by agitter — APPROVED** [review]
Looks good.  I have two minor questions.

File: content/02.main-text.md (hunk 2 above)
Comment (dhimmel) [reply content/02.main-text.md:10]: From the [pandoc docs](http://pandoc.org/MANUAL.html#citation-rendering)

> Citations go inside square brackets and are separated by semicolons: `Blah blah [@smith04; @doe99].`

> You can also write an in-text citation, as follows: `@smith04 says blah.`

How in-text citations are rendered depends on the selected CSL style, I believe.

Comment (agitter) [reply content/02.main-text.md:10]: > How in-text citations are rendered depends on the selected CSL style, I believe.

Got it.  That was the part I wasn't completely sure about.

Thanks for taking care of the merge.

**Pull request merged**